src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/QueueListener.cs

```diff
@@ -73,6 +74,10 @@ internal sealed class QueueListener : IListener, ITaskSeriesCommand, INotificati
             _trace = trace;
             _queueConfiguration = queueConfiguration;
 
+            // if the function runs longer than this, the invisibility will be updated
+            // on a timer periodically for the duration of the function execution
+            _visibilityTimeout = TimeSpan.FromMinutes(10);
```

Comment:
Note that this existing internal value is still not configurable, and I don't think it needs to be.